Source/Core/DolphinQt/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -453,6 +489,18 @@ void EnhancementsWidget::AddDescriptions()
       "scaling filter selected by the game.<br><br>Any option except 'Default' will alter the look "
       "of the game's textures and might cause issues in a small number of "
       "games.<br><br><dolphin_emphasis>If unsure, select 'Default'.</dolphin_emphasis>");
+  static const char TR_OUTPUT_RESAMPLING_DESCRIPTION[] = QT_TR_NOOP(
+      "Affects how the game output image is upscaled or downscaled to the window resolution.<br>"
+      "\"Default\" will rely on the GPU internal bilinear sampler which isn't gamma corrected."
+      "<br>\"Bilinear\" (gamma corrected) is a good compromise between quality and performance."
+      "<br>\"Bicubic\" is smoother than \"Bilinear\"."
+      "<br>\"Hermite\" might offer the best quality when upscaling,"
+      " at a slightly bigger perform cost.<br>\"Catmull-Rom\" is best for downscaling."
+      "<br>\"Nearest Neighbor\" doesn't do any resampling, select if you like a pixelated look."
+      "<br>\"Sharp Bilinear\" works best with 2D games at low resolutions, use if you like a sharp"
+      " look."
+      "<br>\"Box Resampling\" is most expensive but also most accurate downscaling method."
```

Comment:
A comparison to SSAA may be useful here. After all, when we're at the stage of being fine with expensive downsampling, SSAA comes into the picture, and will be the primary competitor to this Box Resampling option.

Reply:
The main difference between box/area resampling and SSAA is that SSAA is not a window resampler. The best example is, say i have a really weird TV, it's 1366x768, and I want the best pixel quality, i can render it at 8k, and Box Resampling will fit the pixels optimally into that weird 1366x768 screen so it looks smooth. This is a weird example and I'm not sure how to fit it into the description.

Reply:
Practically speaking (and weird resolutions discussed above aside), SSAA at the same multiplier as `(internal resolution / display resolution)^2` (e.g. 1080p 4x SSAA vs 4k no AA) will produce superior results because of more complex (compared to axis-aligned grid that high internal resolution gives) sampling pattern typically used by the driver. However, some games suffer from artifacts when SSAA is used, leaving higher internal resolutions as the only viable options.

The bottom line is: high-quality resampling + high internal resolution gives lower image quality at resource consumption very close to comparable SSAA mode, but higher compatibility with games.

Reply:
Resolution scaling is also beneficial when your display resolution or window doesn't match the rendering resolution, which is basically all the times.

Reply:
@v1993 what games have issues with SSAA?

Reply:
I messed around with SSAA a bit. It's my understanding that it works so that the game output resolution stays the same, but internally each pixel was an average of x (e.g. 8) samples instead of 1.

This is great when playing at a low internal resolution (e.g. wii/gc native) in a small or big game window,
but it doesn't actually help at all when playing at a high internal resolution in a window of a <= resolution, because the downscaling was handled poorly, thus all the extra detail would end up lost.
Additionally both MSAA and SSAA cause artifacts in multiple games (e.g. Super Mario Galaxy, Super Mario Sunshine).

Basically SSAA is a "resolution quality scaler" that applies within the rendering phase, while my resampling just deals with the game->window resolution conversion. They aren't really competitors, and 8x SSAA does not look better than 8x internal resolution + area/box downsampling.
As shown in the examples here, SSAA presumably loses a lot of potentially additional detail due it being stored in a lower resolution image.

Native:
![Dolphin_6NrCkMR9Zq](https://github.com/dolphin-emu/dolphin/assets/7011366/54d11cbc-0c53-44c7-bb56-690700eb9af6)

Native + 8x SMAA:
![Dolphin_F9b2JZ8iNs](https://github.com/dolphin-emu/dolphin/assets/7011366/4c354933-55e0-4a09-b467-1df9c9f42651)

8x Native res + Area Resampling:
![Dolphin_x2xVePrD8p](https://github.com/dolphin-emu/dolphin/assets/7011366/34857176-cf5d-4741-bde9-a81c40632c2e)

Reply:
> I messed around with SSAA a bit.

So for best results, we've always told users to configure the Internal Resolution to match their screen resolution (or at least as close as possible), then apply SSAA. Is that what you did when you were experimenting with it?

Reply:
You have an major error in your test: 8x SSAA gives you four samples per pixel, 8x IR gives you 8*8=64 samples per pixel - sample count increases exponentially as resolution grows. To approximate 8x SSAA with IRs you should use `sqrt(8)=2.82...≅3`x IR.

The idea behind comparing box resampling with SSAA is that both effectively result in multiple samples of underlying primitives, providing "true" anti-aliasing. Downsampling has the benefit of higher compatibility (as you've noted for two Mario games, see https://wiki.dolphin-emu.org/index.php?title=Category:Anti-Aliasing_(Config_Required) for some more examples) and while SSAA takes advantage of non-aligned-grid sampling pattern, which allows to eliminate aliasing at much lower performance cost.

Reply:
@MayImilae that's a good suggestion to give to users. I play at 1440p and could barely notice any added quality from SSAA (it actually caused glitches on the pipes in the screenshot), and given I can play at 60fps with a 8x resolution mutliplier + area resampling, I'd personally see no use in SSAA, but I guess it depends on the user and the hardware.

Reply:
@v1993 good point, I didn't think about the different ways the multipliers are applied.
I supposed when comparing x8 SSAA and ~x3 IR, the result is pretty similar. There's not much need in defining which one would generally look best in that case as they still have different purposes.

Reply:
3x IR still looks a lot better than 8x SSAA
![Dolphin_p6C5Ftgc6I](https://github.com/dolphin-emu/dolphin/assets/7011366/810c9932-8936-4807-8580-b0b4aa707f4a)
![Dolphin_nz27R3q0zm](https://github.com/dolphin-emu/dolphin/assets/7011366/314fe198-1732-4116-b65c-bd8b159c255e)
![Dolphin_sD6j6VcTuy](https://github.com/dolphin-emu/dolphin/assets/7011366/d75a5062-6573-4a97-9415-3ef0a1c39a92)

Reply:
Now *that* is impressive. While box resampled picture has a bit more aliasing, many would consider it much more desirable for significantly cleaner look (and you can pump that into some post-processor doing SMAA to smooth out remaining jaggies). Have you tried this with more games, especially those not known for issues with MSAA/SSAA? I'd expect higher IRs to be superior for games utilizing render-to-texture effects, although I'm not sure by how much.

Also, may I ask what card/drivers are those? Exact SSAA/MSAA implementation varies per vendor and possibly even somewhat per graphics API.

Reply:
DX12 RTX 3090